lib/platform/gitlab/index.ts

```diff
@@ -100,28 +100,27 @@ export async function initRepo({
     }
     config.defaultBranch = res.body.default_branch;
     config.baseBranch = config.defaultBranch;
+    if (res.body.http_url_to_repo === null) {
+      throw new Error('no http_url_to_repo found');
+    }
+    logger.debug(`${repository} http URL = ${res.body.http_url_to_repo}`);
+    const repoUrl = URL.parse(`${res.body.http_url_to_repo}`);
```

Comment:
When you have your proxy in front of GitLab, how does GitLab "know" what the proxy's URL for GitLab is? Is it something you need to provision into GitLab yourself?

Reply:
Its a configuration option in gitlab:

`external_url "https://example.com/gitlab"`

It's described in the docs: https://docs.gitlab.com/omnibus/settings/configuration.html#configuring-a-relative-url-for-gitlab